types/styled-components/index.d.ts

```diff
@@ -20,421 +20,423 @@ declare global {
     interface ShadowRoot {}
 }
 
-import * as CSS from 'csstype';
-import * as React from 'react';
-import * as hoistNonReactStatics from 'hoist-non-react-statics';
+declare module 'styled-components' {
```

Comment:
what do u want to solve with this? unless there is some strong reason to wrap the content like this - i would avoid this syntax